### PR TITLE
Add week view for ViewScheduleCommand

### PR DESCRIPTION
**Pull request by limsiying**

### CODE REVIEW

**Comment by limsiying** [comment]
Removed academic year context, for now, to refactor Calendar to LocalDateTime + resolve bugs.

File: src/main/java/Duke.java [hunk 1]
```diff
@@ -45,6 +46,9 @@ private void run() {
                 ui.showError(e.getMessage());
             } finally {
                 ui.endBlock();
+                long end = System.currentTimeMillis();
+                float sec = (end - start) / 1000F;
+                System.out.println(sec + " ms");
```
Comment (mohideenik) [inline src/main/java/Duke.java]: Why are you doing this?

Comment (limsiying) [reply src/main/java/Duke.java]: Ooops i wanted to see how long the command was taking. Will remove!

**Comment by limsiying** [comment]
Closes #48 .

**Pull request merged**